development/cpp/custom_modules_in_cpp.rst

```diff
@@ -449,54 +449,94 @@ There are several steps in order to setup custom docs for the module:
    anything, but we'll be using the ``doc_classes`` name throughout this section.
 
-2. Append the following code snippet to ``config.py``:
+2. Now, we need to edit ``config.py``, add the following snippet:
 
    .. code-block:: python
 
-       def get_doc_classes():
-           return [
-               "ClassName",
-           ]
+        def get_doc_path():
+            return "doc_classes"
 
-       def get_doc_path():
-           return "doc_classes"
+        def get_doc_classes():
+            return [
+                "Summator",
+            ]
+
+The ``get_doc_path()`` function is used by the build system to determine
+the location of the docs. In this case, they will be located in the 
+``modules/summator/doc_classes`` directory. If you don't define this,
+the doc path for you module will fall back to the main ``doc/classes`` 
+directory.
 
 The ``get_doc_classes()`` method is necessary for the build system to
-know which documentation classes of the module must be merged, since the module
-may contain several classes. Replace ``ClassName`` with the name of the class
-you want to write documentation for. If you need docs for more than one class,
-append those as well.
+know which registered classes belong to the module. You need to list all of your 
+classes here. The classes that you don't list will end up in the 
+main ``doc/classes`` directory.
 
-The ``get_doc_path()`` method is used by the build system to determine
-the location of the docs. In our case, they will be located in the ``doc_classes``
-directory.
+.. tip::
+
+    You can use git to check if you have missed some of your classes by checking the 
+    untracked files with ``git status``. For exmample::
+
+        user@host:~/godot$ git status
+
+    Example output::
 
-3. Run command:
+        Untracked files:
+            (use "git add <file>..." to include in what will be committed)
+            
+            doc/classes/MyClass2D.xml
+            doc/classes/MyClass4D.xml
+            doc/classes/MyClass5D.xml
+            doc/classes/MyClass6D.xml
+            ...
+    
+
+3. Now we can generate the documentation:
+
+We can do this via running godot's doctool e.g. ``godot --doctool <path>``,
+which will dump the engine API reference to the given ``<path>`` in XML format.
+
+In our case we'll point it to the root of the cloned repository. You can point it
+to an another folder, and just copy over the files that you need.
+
+Run command:
 
    ::
 
-      godot --doctool <path>
+      user@host:~/godot$ ./bin/godot* --doctool .
 
-This will dump the engine API reference to the given ``<path>`` in XML format.
-Notice that you'll need to configure your ``PATH`` to locate Godot's executable,
-and make sure that you have write access rights. If not, you might encounter an
-error similar to the following:
+.. note::
 
-.. code-block:: console
+    If you want, you can prevent the core classes to be generated with the 
```

Comment:
```suggestion
    If you want, you can prevent the core classes from being generated with the 
```

Reply:
But I'd straight out remove this note, it's not relevant for this use case.
What it does it that it doesn't dump documentation for Variants, but it still includes everything else from the API, so:
- Its actual use case is dubious
- It's not relevant for writing module docs

Reply:
Last time I tested `--no-docbase` will also remove all base docs as well, polluting the `git status` with deleted classes, but this something which should be fixed IMO.

Reply:
I removed this one.